### DISCUSSION TITLE
Macros

### DISCUSSION THREAD

**Discussion by ThatXliner** [atom/atom#23272] [Ideas]
We need macros.

Like actually well-supported macros.

Record DOM keyevents, whatever, I just want a working macro package.

All the other ones don't work.

**Comment by Tanapl02** [comment]
> We need macros.
> 
> Like actually well-supported macros.
> 
> Record DOM keyevents, whatever, I just want a working macro package.
> 
> All the other ones don't work.

Start

**Reply by Tanapl02** [reply]
Start
